src/actions/LoginActions.tsx

```diff
@@ -53,8 +53,10 @@ export function initializeAccount(navigation: NavigationBase, account: EdgeAccou
     const syncedSettings = await readSyncedSettings(account)
     const { walletsSort } = syncedSettings
     dispatch({ type: 'LOGIN', data: { account, walletSort: walletsSort } })
-    await dispatch(loadAccountReferral(account))
-    const newAccount = account.newAccount
+    const { newAccount } = account
+    if (newAccount) {
+      await dispatch(loadAccountReferral(account))
+    }
```

Comment:
This will break Peter's analytics. We need to load this information on every login, but we only need to *await* the load for new accounts. Instead, I would propose:

```ts
const { newAccount } = account
const referralPromise = dispatch(loadAccountReferral(account))

if (newAccount) {
  await referralPromise
  ... plus the rest of the wallet-creation stuff from the existing if statement right below ...

} else {
  navigation.push('edgeApp', {})
  // Insert this as a new line in the existing `else` statment:
  referralPromise.catch(() => { /* silently fail - we just get worse analytics */ })
}
```